app/src/main/res/layout/activity_filter_shelters.xml

```diff
@@ -1,135 +1,143 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.v4.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is there a reason why this is a `NestedScrollView` and not just a standard `ScrollView`?

Reply:
Didn't show up in autocomplete.
Can change if functionality is preserved.

Reply:
It shows up for me in autocomplete (the tag is just `<ScrollView>`). Nested scroll view is probably fine but is meant for when its a scroll view inside of a scroll view, which causes the issues we're having on the shelter list so may be better to use a simple `ScrollView`

Reply:
Strange, maybe I need to have intellij rebuild its autocomplete dict.
I'm all for less problems, let's make the change.